src/libraries/System.Collections.Concurrent/tests/BlockingCollectionTests.cs

```diff
@@ -744,6 +744,7 @@ public static void TestAddAnyTakeAny_Longrunning(int numOfAdds, int numOfTakes,
         [Theory]
         [InlineData(4, 2048, 2, 64)]
         [OuterLoop]
+        [PlatformSpecific(~TestPlatforms.Browser)] // Cannot wait on monitors on this runtime.
```

Comment:
AFAIR we used to add `PlatformDetection.IsThreadingSupported` for case ` Cannot wait on monitors on this runtime.`:

`[ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]`.

Though, not sure if it's still so.

Reply:
Thanks Maxim

Reply:
These should all be modified now to use `[ConditionalXXX(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]`.

Thanks for the heads up on this @MaximLipnin

Reply:
Does `IsThreadingSupported` capture the meaning? Or is it more about locking? We have lots more cases of "threading" than the ones here in this PR which do not have this conditional attribute.